Source/PeleC.H

```diff
@@ -691,6 +693,13 @@ protected:
 
   static amrex::Vector<int> src_list;
 
+  static bool use_typ_vals_chem;
+  static bool use_typ_vals_chem_usr;
+  static amrex::Real typical_rhoY_val_min;
+  static int reset_typical_vals_int;
+  amrex::Vector<amrex::Real> typical_values_chem;
+  static amrex::Vector<amrex::Real> typical_values_chem_usr;
```

Comment:
feels like only one of these vectors is truly necessary. Depending on the type of typical values required, it will be filled differently? Suggesting this because these modes look exclusive so we can use the same vector.

Reply:
again, two vectors are needed with the assumption that typical_values_chem is needed at each level. Now that I think of it, I may not need class member variable. Will change this.